src/h5web/dimension-mapper/SlicingSlider.module.css

```diff
@@ -4,7 +4,7 @@
   flex-direction: column;
   padding: 0 0.5rem;
   --thumb-height: 1.5rem;
-  --mark-height: 0.4375rem;
+  --mark-height: 0.5rem;
```

Comment:
So the marks are horizontally centred on the track. (There was a 1px misalignment.)